tests/integration/test_snap.py

```diff
@@ -237,3 +237,16 @@ def test_reset_hold_refresh():
     snap.hold_refresh(0)
     result = check_output(["snap", "refresh", "--time"])
     assert "hold: " not in result.decode()
+
+
+def test_alias():
+    cache = snap.SnapCache()
+    lxd = cache["lxd"]
+    lxd.alias("lxc", "testlxc")
+    result = check_output(["snap", "aliases"])
+    found_alias = False
+    for line in result.decode().split("\n")[1:]:
+        if ["lxd.lxc", "testlxc", "manual"] == [val for val in line.split(" ") if val]:
```

Comment:
`line.split()` (without args) is [equivalent](https://docs.python.org/3/library/stdtypes.html#str.split) to `[val for val in line.split(" ") if val]`.

Also, the order of the equality test is a bit confusing ([Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) are uncommon in Go).

Could potentially simplify and remove the found flag by using `any(<generator expression>)`. Also, it would be good to include the output in the failure message to help debug any issues when they occur:

```python
found = any(line.split() == ["lxc.lxc", "testlxc", "manual"]
            for line in result.splitlines())
assert found, result
```

Reply:
Thanks for the suggestion. Updated